openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/RF2/certificate_estimation/RF2_ESC_calculation.py

```diff
@@ -136,14 +136,31 @@ class RF2_ESC_calculation(Variable):
     label = 'The number of ESCs for HVAC1'
 
     def formula(buildings, period, parameters):
-      RF2_electricity_savings = buildings('RF2_electricity_savings', period)
+      electricity_savings = buildings('RF2_electricity_savings', period)
       electricity_certificate_conversion_factor = 1.06
+      replacement_activity = buildings('RF2_replacement_activity', period)
+      EEI_eligible_replacement = buildings('RF2_product_EEI_ESC_replacement_eligibility', period)
+      EEI_eligible_install = buildings('RF2_product_EEI_ESC_install_eligibility', period)
+
+      RF2_eligible_ESCs = np.select(
+            [
+                replacement_activity * EEI_eligible_replacement,
+                np.logical_not(replacement_activity) * np.logical_not(EEI_eligible_replacement),
+                replacement_activity * EEI_eligible_install,
+                np.logical_not(replacement_activity) * EEI_eligible_install
```

Comment:
we should also add 

false replacement_activity and false EEI_eligible_install

Reply:
I think we should add all combinations for the three variables

Reply:
Adding these extra conditions is breaking the tests, so we are not going to do this.